Sources/SentryCrash/Recording/Tools/SentryCrashUUIDConversion.h

```diff
@@ -0,0 +1,56 @@
+#ifndef SENTRY_SENTRYCRASHUUIDCONVERSION_H
+#define SENTRY_SENTRYCRASHUUIDCONVERSION_H
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#include <stdbool.h>
+
+/** Converts SentryCrashBinaryImage.uuid to a human readable 36 charactacters long hex
+ * representation.
+ *
+ * @param src The pointer to an UUID
+ *
+ * @param dst A buffer with the length of 37 to hold the human readable UUID.
+ *
+ */
+static void
+sentrycrashdl_convertBinaryImageUUID(const unsigned char *src, char *dst)
```

Comment:
`M`: Keep in mind, we shouldn't lose this function once we move away from KSCrash

Reply:
I'm not sure if we need to keep this, since it is very specific to KSCrash right now. That's why I also named it like this. Anyway, the tests are going to fail if we mess it up, so we should be good 😁